one_fm/templates/pages/applicant-docs.js

```diff
@@ -1,85 +1,168 @@
+var is_kuwaiti = 0;
+function front_civil_extract(input){
+    let file = input.files[0];
+
+    let reader = new FileReader();
+
+    reader.readAsDataURL(file);
+
+    reader.onload = function() {
+      front_civil = reader.result;
+      front_civil = front_civil.replace(/^data:image\/\w+;base64,/, "");
+    };
+
+    reader.onerror = function() {
+      console.log(reader.error);
+    };
+
+};
+
+function back_civil_extract(input){
+    let file = input.files[0];
+    let reader = new FileReader();
+
+    reader.readAsDataURL(file);
+
+    reader.onload = function() {
+      back_civil = reader.result;
+      back_civil = back_civil.replace(/^data:image\/\w+;base64,/, "");
+    };
+
+    reader.onerror = function() {
+      console.log(reader.error);
+    };
+};
+
+function front_passport_extract(input){
+  let file = input.files[0];
+  let reader = new FileReader();
+
+  reader.readAsDataURL(file);
+
+  reader.onload = function() {
+    front_passport = reader.result;
+    front_passport = front_passport.replace(/^data:image\/\w+;base64,/, "");
   };
 
-  if (window.localStorage.getItem("linkedIn")) {
-      getLinkedInData();
-  }
+  reader.onerror = function() {
+    console.log(reader.error);
+  };
+};
+
+function back_passport_extract(input){
+  let file = input.files[0];
+  let reader = new FileReader();
+
+  reader.readAsDataURL(file);
+
+  reader.onload = function() {
+    back_passport = reader.result;
+    back_passport = back_passport.replace(/^data:image\/\w+;base64,/, "");
+  };
+
+  reader.onerror = function() {
+    console.log(reader.error);
+  };
+};
+function test(input){
+var date = input.value;
+console.log(date)
 }
-else {
-    alert("Please Signup or Login!");
-    window.location = "applicant-sign-up";
+
+function populate_nationality(){
+  frappe.call({
+    type: "GET",
+    method: "one_fm.templates.pages.applicant-docs.populate_nationality",
+    callback: function(r) {
+      langArray = r.message;
+      console.log(langArray)
+      if(langArray){
+        var select = document.getElementById("nationality");
+        for (let i=0; i<=langArray.length;i++) {
+          select.options[select.options.length] = new Option(langArray[i], langArray[i]);
+     }
+
+      }
+    }
+  }); 
 }
-const skipSignup = () => {
-    if(localStorage.getItem("currentEasyJobOpening"))
-        window.location = "easy_apply";
-    else
-        window.location = "job_application";
 
+function fetchNationality(code){
+  frappe.call({
+    type: "GET",
+    method: "one_fm.templates.pages.applicant-docs.fetch_nationality",
+    args: {code :code},
+    callback: function(r) {
+      if(r.message){
+        document.getElementById("nationality").value = r.message;
+      }
+      else{
+        document.getElementById("nationality").value = "";
+      }
+      
+    }
+  }); 
 }
+function upload(){
+  civilid_check = document.getElementById("noCivilID").checked;
+  //civilid_check = document.getElementById("noPassport").checked;
```

Comment:
- Remove commented codes

Reply:
@pjamsheer , I needed the Commented Code for the next story. So, I kept it.